tests/Feature/ColorControllerTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
@rajan-parmar Do we plan to cover any more functionalities?

Reply:
> @rajan-parmar Do we plan to cover any more functionalities?

based on functionality it's ok for now. for testing others are pending.